runbotics-orchestrator/src/main/java/com/runbotics/repository/ProcessRepository.java

```diff
@@ -38,6 +38,12 @@ public interface ProcessRepository extends JpaRepository<Process, Long>, JpaSpec
     )
     Page<Process> findByBotCollection(String collectionName, String userLogin, Pageable pageable);
 
+    @Query(
+        value = "SELECT COUNT(process.id) FROM process WHERE process.created_by_id = :id",
+        nativeQuery = true
+    )
```

Comment:
try removing nativeQuery. In case we want to use another DB (like MySQL instead of PostgreSQL, which is slightly probable, but not impossible) this will no longer work

Reply:
if the language syntax is different e.g. in T-SQL there's TOP but in MySQL there's LIMIT